builder/builder.go

```diff
@@ -134,7 +135,7 @@ func (b *Builder) InitMix(upstreamVer string, mixVer string, allLocal bool, allU
 	// Deprecate '.clearurl' --> 'upstreamurl'
 	if _, err := os.Stat(filepath.Join(b.Config.Builder.VersionPath, ".clearurl")); err == nil {
 		b.UpstreamURLFile = ".clearurl"
-		log.Println("Warning: '.clearurl' has been deprecated. Please rename file to 'upstreamurl'")
+		log.Warning(log.Mixer, "Warning: '.clearurl' has been deprecated. Please rename file to 'upstreamurl'")
```

Comment:
Let's not pass the string with the term "Warning: " to log.Warning(). It will lead to redundant terms in the log file because of the "WRN" annotation.
Instead lets modify the log.Warning() to prepend the term "Warning: " to any string that is passed to it.

Reply:
yes, will update the string.